src/dispatch/case/flows.py

```diff
@@ -529,10 +531,38 @@ def case_to_incident_endpoint_escalate_flow(
     case.escalated_at = datetime.utcnow()
     case.status = CaseStatus.escalated
 
+    incident = incident_service.get(db_session=db_session, incident_id=incident_id)
+
+    channel_case = True if case.conversation.thread_id else False
```

Comment:
You return True if it's a thread based conversation, which I find it a little bit confusing when reading the name of the variable `channel_case`
```suggestion
    channel_case = False if case.conversation.thread_id else True
```
or
```
channel_case = True if not case.conversation.thread_id else False
```

Reply:
Yeah sorry, it's not ready for review yet -- locally I change this to a property on the Case model so it's more intuitive:

```python
    @property
    def has_channel(self) -> bool:
        return True if not self.conversation.thread_id else False

    @property
    def has_thread(self) -> bool:
        return True if self.conversation.thread_id else False
```

Reply:
Got it. I'll wait until it's ready then :)